docs/reference/run.md

```diff
@@ -667,6 +667,17 @@ It also causes any seccomp filters to be applied later, after privileges have be
 which may mean you can have a more restrictive set of filters.
 For more details, see the [kernel documentation](https://www.kernel.org/doc/Documentation/prctl/no_new_privs.txt).
 
+## Specifying an init process
+
+You can use the `--init` or `--init-path` flags to specify a process which
+should be used as the PID 1 in the container, and which binary to use for this
+init process, respectively. Specifying an init process ensures the usual
+responsibilties of an init system, such as reaping zombie processes, are
+performed inside the created container.
+
+The default init process used when specifying `--init` without `--init-path` is
```

Comment:
Maybe:

The default `init` process used when `--init-path` is not specified is the first `docker-init` executable found in the path. To check this, run `which docker-init`. The default `docker-init` binary is provided by [tini](link-to-tini){: target="_blank" class="_" }

(all of that makes the link to tini open in a new window, without wrecking your text editor's notion of underscores meaning italics)